lib/data/entity.js

```diff
@@ -354,20 +354,32 @@ const ConflictType = {
   SOFT: 'soft', // discrepancy in version number but no overlaping properties
   HARD: 'hard' // discrepancy in version number and one/more properties updated simultaneously
 };
-const getWithConflictDetails = (defs) => {
+const getWithConflictDetails = (defs, audits, relevantToConflict) => {
   const defMap = new Map(defs.map(d => [d.version, d]));
 
   const result = [];
 
+  const lastResolveEvent = audits.findLast(a => a.action === 'entity.update.resolve');
+
+  const auditMap = new Map(audits.map(a => [a.details.entityDefId, a.details]));
```

Comment:
There can be multiple audit log events with the same `entityDefId`, right? For example, if you cause a conflict, then you resolve the conflict. But I guess all the relevant details would be the same among those multiple events, right? Just making sure I understand how this works.

Reply:
For the safe side, I have filtered the audits for only `create` and `update` events